infra/src/main/kotlin/fi/fta/geoviite/infra/geography/GeoToolsGeometries.kt

```diff
@@ -41,7 +41,8 @@ fun calculateDistance(points: List<IPoint>, ref: CoordinateReferenceSystem): Dou
         }
 }
 
-fun crs(srid: Srid): CoordinateReferenceSystem = CRS.decode(srid.toString())
+private val crsCache: MutableMap<Srid, CoordinateReferenceSystem> = mutableMapOf()
+fun crs(srid: Srid): CoordinateReferenceSystem = crsCache.getOrPut(srid) { CRS.decode(srid.toString()) }
```

Comment:
Kakutetaan crs-luonnit muistiin, sillä kyseessä on melko kallis operaatio. Varsinkin julkaisulokissa näitä tehdään aika paljon, joten tämä nopeuttaa asioita.